pkg/harbor/members.go

```diff
@@ -70,20 +112,14 @@ type projectMemberRequestBody struct {
 	MemberUser  *userEntity `json:"member_user"`
 }
 
-// type projectUserMember struct {
-// 	Name      string
-// 	Role      role
-// 	projectId int
-// }
-
-func (p *projectAPI) getUserMembers(projectID int) ([]*projectMemberEntity, error) {
+func (p *projectAPI) getMembers(projectID int) ([]*projectMemberEntity, error) {
+	// FIX: thread unsafe handling of parsedUrl
 	p.reg.parsedUrl.Path = fmt.Sprintf("%s/%d/members", path, projectID)
 	p.reg.logger.V(1).Info("creating new request", "parsedUrl", p.reg.parsedUrl.String())
 	req, err := http.NewRequest(http.MethodGet, p.reg.parsedUrl.String(), nil)
```

Comment:
Not really related to this particular PR, but here I started wonder if we are setting any timeouts for our HTTP requests (which would be super-important especially once this will be promoted to an operator). It seems not? https://github.com/kubermatic-labs/registryman/blob/master/pkg/harbor/registry.go#L55 `http.DefaultClient` does not set any.

Reply:
We are not setting HTTP client timeouts at this point. We will definitely need this in the future.